src/riak_kv_stat.erl

```diff
@@ -263,6 +263,11 @@ do_update({put_fsm_time, Bucket,  Microsecs, Stages, PerBucket, CRDTMod}) ->
     ok = create_or_update([P, ?APP, node, puts, Type, time], Microsecs, histogram),
     ok = do_stages([P, ?APP, node, puts, Type, time], Stages),
     do_put_bucket(PerBucket, {Bucket, Microsecs, Stages, Type});
+do_update({index_fsm_time, Microsecs, ResultCount}) ->
+    P = ?PFX,
```

Comment:
Correct, but I like 3 copies of ?PFX in arguments below better... Renaming a macro makes it not as easy to see that a macro is used.

In line 272 below the macro is also used, so more in line with that code

Reply:
The convention in the module is that if ?PFX is required more than once in a  function, it is called then assigned rather than repeatedly called.  It might be a performance thing (this a very active piece of code, ignoring excess calls to get_env), or perhaps a consistency thing (so that the prefix can't change across a set of updates.

Without knowing why this is the convention, I don't wish to change it.